### PR TITLE
Add a test for configuration round trip

### PR DESCRIPTION
**Pull request by Vlatombe**
Split from #179 as this requires a core bump.

### CODE REVIEW

**Comment by Vlatombe** [comment]
@reviewbybees

File: src/test/java/org/jenkinsci/plugins/mesos/MesosCloudTest.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+import static org.junit.Assert.assertTrue;
+
+public class MesosCloudTest {
+
+    @Rule
+    public JenkinsRule r = new JenkinsRule();
+
+    @Test
+    public void configRoundTrip() throws Exception {
```
Comment (vinodkone) [inline src/test/java/org/jenkinsci/plugins/mesos/MesosCloudTest.java]: what is this testing? could use some comments here.

File: pom.xml [hunk 2]
```diff
@@ -3,7 +3,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.565.3</version><!-- the minimum Jenkins core compatible version to run plugin on -->
+    <version>1.627</version><!-- the minimum Jenkins core compatible version to run plugin on -->
```
Comment (vinodkone) [inline pom.xml]: is this an old enough version? also, needs an email to the mailing list.

Comment (Vlatombe) [inline pom.xml]: It's probably too recent, I'd wait to have a new LTS more recent to merge this. This is mostly to record the minimal version for this to work.

**Comment by Vlatombe** [comment]
No core bump required anymore as the test-harness version can be set independently.

**Comment by rsandell** [comment]
:bee:

**Comment by Vlatombe** [comment]
Relaunching the build to check if the timeout is just a one-time thing

**Comment by Vlatombe** [comment]
Tests hang with JDK7, but pass with JDK8... Don't know why.

**Pull request closed without merging**